### DISCUSSION TITLE
Camera P1P/P1S not updating

### DISCUSSION THREAD

**Discussion by Brainiac1986** [greghesp/ha-bambulab#366] [Q&A]
Hey i have a question yesterday i installed the repository on my HA instance and loaded my printer in

only the issue is the camera image doesn't update and keeps standing on same picture during installation.
i used the cloud method and getting data directly from printer and not from cloud

**Comment by AdrianGarside** [comment]
Can you check the debug log? And which version of the integration are you using? I just released v2.0.3 yesterday and that had some reliability fixes for the P1/A1 camera image.

**Comment by AdrianGarside** [comment]
No  response and this should have been an issue not a discussion.
